website/docs/r/is_vpc_routing_table.html.markdown

```diff
@@ -51,11 +52,11 @@ Review the argument references that you can specify for your resource.
 In addition to all argument reference list, you can access the following attribute reference after your resource is created.
 
 - `href` - (String) The routing table URL.
-- `id` - (String) The routing table ID. The ID is composed of `<vpc_id>/<vpc_route_table_id>` of the VPC route.
+- `id` - (String) The generated resource ID. The ID is composed of `<vpc_id>/<vpc_routing_table_id>` of the VPC routing table.
```

Comment:
instead of generated resource, we can mention (like in other child resource docs)
"The unique ID/Identifier of the routing table. The ID is composed of `<vpc_id>/<vpc_routing_table_id>`"

of the VPC routing table can be removed i guess as the ID form is self explanatory